jupyter-extension/README.md

```diff
@@ -1,5 +1,7 @@
 # jupyterlab-pachyderm
 
+blah
```

Comment:
lol whats this?

Reply:
This is just to touch the Jupyter folder so the right CI runs, I have a note to eliminate before merge